replace_string_value_example_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2022 Michael J. Fromberger. All Rights Reserved.
+
+package tomledit_test
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/creachadair/tomledit"
+	"github.com/creachadair/tomledit/transform"
+)
+
+func Example() {
+	data := []byte(`
+	A = "Va"
+
+	[ B ]
+	C = "Vc"
+
+	[ D.E ]
+	F = "Vf"
+	`)
+	replaceList := []struct {
```

Comment:
I don't know that we really need a table for an example, let's just pick one of the more interesting cases like "D.E.F" to "vf2" or something. The reader can extrapolate how to do it for multiple cases from that.